testsuite/Makefile

```diff
@@ -21,7 +21,10 @@ FIND=find
 ROOTDIR = ..
 include $(ROOTDIR)/Makefile.common
 
-OCAMLTESTDIR_CYGPATH=$(shell $(CYGPATH) $(BASEDIR)/$(DIR)/_ocamltest)
+BASEDIR_TARGET := $(shell $(CYGPATH) "$(BASEDIR)")
+ROOTDIR_TARGET := $(BASEDIR_TARGET)/$(ROOTDIR)
```

Comment:
I don't understand the naming convention, why are those things named with `_TARGET`? Is there some sort of cross-execution in mind?

Reply:
Indeed, but it's not in mind - there _is_ cross-execution, but I used the wrong suffix, as it should be `_HOST`. For the Windows builds, we're building on Cygwin for execution on a native Windows host - i.e. it already is cross-compilation, we just don't feel it because the "build" system is capable of executing host binaries (although we do feel it, because the build and host systems have differing views of filenames...)

I changed it because `ROOTDIR_CYGPATH` and `BASEDIR_CYGPATH` felt strange for systems where `CYGPATH` is just `echo` (i.e. all the normal systems...)